### DISCUSSION TITLE
Best practices for decentralized blogging and websites

### DISCUSSION THREAD

**Discussion by bmann** [memser-spaceport/pl-network-hub#15] [Ideas]
We still see a lot of people across the PLN using centralized systems like Medium or Substack.

Let's gather together some good options, how tos, and getting started kits so everyone from individuals to organizations and projects across the PLN can easily post to the DWeb.

Some tools to recommend might be like Ghost (instead of Substack or Medium). This could be a good thing for the PLN Marketing Council to look at cc @heybeckyv

**Comment by pandeyanujk** [comment]
@bmann interesting idea! Adding @DeniceZ so she can weigh in & add others from the marketing council to weigh in.

**Reply by pandeyanujk** [reply]
+ @rschanger. What's your thought? Any teams within the network you are tracking they may also be worth bringing to this convo?

**Comment by DeniceZ** [comment]
We were talking about this in a Marketing Council meeting last month actually when discussing [SEO for web3](https://coda.io/d/PLN-Councils_d6eF5JClyXP/SEO-101_suveN#_lurSH). Mirror was mentioned but I'm not actually familiar with Ghost.

I think there's a tradeoff between what is easily accessible and trackable in centralized systems vs the time to setup and value add of decentralized systems. Aside from the benefit of being decentralized, there needs to be a clearer reason for someone to not pick a centralized option, and in our discussion back then it really wasn't clear. I'd be curious to hear people's thoughts on what they've gotten out of options like Ghost that is 'better' than Medium or Substack

**Reply by heybeckyv** [reply]
Thanks for getting this kicked off in the PLN chat, Denice! I've mentioned headless Ghost and Blockenberg. I'm also curious to hear what people's thoughts are! I believe that if we promote decentralized tech, we should use it! It also won't lock marketers into a specific platform if they decide they want to switch services (b/c you know that is a painful process).

**Reply by bmann** [reply]
@DeniceZ Mirror publishes to Arweave, a direct competitor to our protocols. 

Ghost is web2, but can be used to publish to IPFS. 

Why is it better than Medium or Substack? It's open source, it's not a private commercial platform, we could (as per my other thread) enable 3000 blogs on IPFS for all PLN with it.

The longer answer is: what are we all doing here if we just use commercial web2 tools? We should be more aligned with our principles. 

Like I totally get that Marketing needs to also post where people are. We post on Twitter today still, but are actively posting into ActivityPub and BlueSky. There are Filecoin and IPFS Accounts on BlueSky already which is great to see. 

I'm suggesting that Marketing and other teams get support in selecting tools that align with our principles in a repeatable way.

As always, I'm just one voice putting things in the Idea Hub here. Is this the most important thing to work on? Probably not, but let's see who is interested and who might upvote this. 

I really hope to see more ideas on what teams across the PLN want and/or might be interested in collaborating on.

**Reply by rschanger** [reply]
Hi @bmann! 

Good questions! Don't know the solution. Wish I had time to explore more. I don't know Ghost. I really like Bluesky.  

And I'm remiss that we're not more aligned with web3 tech, though I do use Huddle01 for one weekly 1:1.

I think back to when I started using SoundCloud or IG — I was an early user for both because a close friend was at USV. They didn't have massive user base at the time. But the reason I continued to use them was because there was a novel and alluring utility in both that didn't exist before.

So, maybe where we begin using tools that align with our web3 principles (and I'm talking about work scenarios, where resources are valuable) is when the tool does something new and fills a use gap (that we can further exploit for the wicked world of marketing :).

Maybe Bluesky becomes that — ultimately users determine what the platform is good for — though I hope it doesn't get overrun by marketing, and we can find a use case that doesn't exist at the moment. 

Anyway, just a thought starter — welcome any and all POV, comments criticism.
